Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -347,6 +353,37 @@ namespace AzToolsFramework
     {
         Clear();
 
+        // determine whether this node should be expanded
+        auto forceExpandAttribute = domArray.FindMember(AZ::Dpe::Nodes::Row::ForceAutoExpand.GetName());
```

Comment:
Could also do: `AZ::Dpe::Nodes::Row::ForceAutoExpand.ExtractValueFromNode(domArray).value_or(false)` (same below with `AutoExpand`)

Reply:
fixed! Make sure it's to your liking